### PR TITLE
Feature: Add mapping for Reports to from NetSuite to Namely in User Experience

### PR DESCRIPTION
**Pull request by cored**
Fix: https://namely.atlassian.net/browse/CORE-88

### CODE REVIEW

File: app/models/net_suite/normalizer.rb [hunk 1]
```diff
@@ -89,6 +90,12 @@ def subsidiary
       { "internalId" => @subsidiary_id }
     end
 
+    def supervisor
+      {
+        "internalId" => @attributes.fetch("internalId", Fields::NullValue.new).to_s
```
Comment (bobbytables) [inline app/models/net_suite/normalizer.rb]: So this logic isn't quite right I think. It needs to be the internalId of the supervisor, not the current profile being exported.

Comment (cored) [inline app/models/net_suite/normalizer.rb]: I see, I'll check thst out.

File: spec/models/net_suite/normalizer_spec.rb [hunk 2]
```diff
@@ -24,7 +26,8 @@
         subsidiary
         releaseDate
         nullFieldList
-      ))
+        supervisor
+                                                    ))
```
Comment (poopoothegorilla) [inline spec/models/net_suite/normalizer_spec.rb]: this looks wierd haha

Comment (bobbytables) [inline spec/models/net_suite/normalizer_spec.rb]: I agree.

File: spec/models/net_suite/normalizer_spec.rb [hunk 3]
```diff
@@ -178,6 +181,25 @@
...
+        ).to eq("internalId" => "456")
+      end
+
+      it "returns a blank without a netsuite_id" do
+        export_attributes = export
+
+        expect(
+          export_attributes["supervisor"]
+        ).to eq("internalId" => nil)
+      end
```
Comment (bobbytables) [inline spec/models/net_suite/normalizer_spec.rb]: We may need to validate if this breaks when we send the data over to NetSuite.

Comment (bobbytables) [inline spec/models/net_suite/normalizer_spec.rb]: The nil value to be specific.

Comment (cored) [inline spec/models/net_suite/normalizer_spec.rb]: @bobbytables got it

**Pull request closed without merging**